parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnWriterV1.java

```diff
@@ -74,10 +74,10 @@ public ColumnWriterV1(
     this.valueCountForNextSizeCheck = INITIAL_COUNT_FOR_SIZE_CHECK;
     resetStatistics();
 
-    ParquetProperties parquetProps = new ParquetProperties(dictionaryPageSizeThreshold, writerVersion, enableDictionary);
+    ParquetProperties parquetProps = new ParquetProperties(dictionaryPageSizeThreshold, writerVersion, enableDictionary, pageWriter.getAllocator());
```

Comment:
instead of looking up the allocator for the pageWriter we should be injected the allocator as a constructor parameter.